### PR TITLE
CLOUDSTACK-8895: Verify if storage on storage pool can be attached to VM

### PR DESCRIPTION
**Pull request by pritisarap12**
Test case to verify if data volume uploaded in a storage pool(Cluster wide storage pool) is available for attachment to a Virtual Machine.and also check that after attachment the volume is in correct storage pool.

### CODE REVIEW

**Comment by pritisarap12** [comment]
Test case result:

Attach Data Disk on CWPS To VM ... === TestName: test_01_attach_datadisk_to_vm_on_zwps | Status : SUCCESS ===
ok

----------------------------------------------------------------------
Ran 1 test in 188.574s

OK

**Comment by runseb** [comment]
@pritisarap12 can this be run via simulator ?

**Comment by pvr9711** [comment]
@runseb, this cannot be run via simulator.  
LGTM

**Comment by remibergsma** [comment]
@pvr9711 Where is your LGTM based on? I'd like to see some background, as otherwise we don't know what you tested / reviewed.

**Comment by sanju1010** [comment]
I did a code walk-through and looks good.
LGTM !!

File: test/integration/testpaths/testpath_attach_disk_zwps.py [hunk 1]
```diff
@@ -207,3 +208,188 @@ def test_01_attach_datadisk_to_vm_on_zwps(self):
             "Check: Data if Disk is attached to VM")
 
         return
+
+
+class TestAttachDataDiskOnCWPS(cloudstackTestCase):
```
Comment (DaanHoogland) [inline test/integration/testpaths/testpath_attach_disk_zwps.py]: is the name of this class a typo? it ends with CWPS (cluster wide primary storage?).

**Comment by sanju1010** [comment]
It is not type Daan. First test inside TestAttachDataDisk class tests some functionality on ZWPS and this one is for CWPS.
Test Results :

Attach Data Disk To VM on ZWPS ... === TestName: test_01_attach_datadisk_to_vm_on_zwps | Status : SUCCESS === 
ok 
---------------------------------------------------------------------- 
Ran 1 tests in 122.641s 
OK

LGTM !!

**Comment by sanju1010** [comment]
@DaanHoogland / @remibergsma can you please merge this PR?

**Comment by DaanHoogland** [comment]
@sanju1010 it shouldn't be in the same file then, or the file should be renamed to a more generic name

**Pull request merged**